test/plugin/test_out_google_cloud.rb

```diff
@@ -839,6 +877,32 @@ def test_multiple_container_logs_metadata_from_tag
     end
   end
 
+  def test_one_cloudfunctions_log
+    setup_gce_metadata_stubs
+    setup_cloudfunctions_metadata_stubs
+    setup_logging_stubs
+    d = create_driver(APPLICATION_DEFAULT_CONFIG, CLOUDFUNCTIONS_TAG)
+    d.emit(cloudfunctions_log_entry(0))
+    d.run
+    verify_log_entries(1, CLOUDFUNCTIONS_PARAMS)
+  end
+
+  def test_multiple_cloudfunctions_logs
+    setup_gce_metadata_stubs
+    setup_cloudfunctions_metadata_stubs
+    setup_logging_stubs
+    d = create_driver(APPLICATION_DEFAULT_CONFIG, CLOUDFUNCTIONS_TAG)
+    [2, 3, 5, 11, 50].each do |n|
+      # The test driver doesn't clear its buffer of entries after running, so
+      # do it manually here.
+      d.instance_variable_get('@entries').clear
+      @logs_sent = []
+      n.times { |i| d.emit(cloudfunctions_log_entry(i)) }
+      d.run
+      verify_log_entries(n, CLOUDFUNCTIONS_PARAMS)
+    end
+  end
+
```

Comment:
Add tests for the cases where running_cloudfunctions is true but the tag or log do not match the regex